### PR TITLE
HTML Client for Web Services Using DHTML Behavior

### LINKED ISSUE #3349: HTML Client for Web Services using DHTML Behavior
### Proposed title of article

HTML Client for Web Services using DHTML Behavior

### Introduction paragraph (2-3 paragraphs):

DHTML makes a webpage dynamic, but so does Javascript DHTML allows us to change the look, content, and style of a webpage after it has been loaded without changing or deleting anything that is already on the browser's webpage.

In this article, we will learn how DHTML can change the content of a webpage on demand without requiring the browser to erase everything else. We will learn how to create a web service as well as an HTML client.

### Key takeaways:
1. The reader will learn how to create the web service.
2. The reader will learn how to create the HTML client.
3. The leader will learn how to test web service and HTML client.

### References: 
N/A

### PR DESCRIPTION
**Pull request by borosteve**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #3349 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Good afternoon and thank you for submitting your article for publishing. @borosteve 
Your article has been entered into our queue and should be reviewed (and published pending any changes) as soon as a peer reviewer is finished with the ones in the queue before it. :rocket:
Thank you.

**Comment by hectorkambow** [comment]
plagiarism check done - article will benefit from going over the % of red flags. 

[Html client for web services using dhtml behavior #3636.pdf](https://github.com/section-engineering-education/engineering-education/files/7117870/Html.client.for.web.services.using.dhtml.behavior.3636.pdf)

**Comment by borosteve** [comment]
Hi, @hectorkambow 
I have made the changes you requested. It is now okay.
Thank you.

**Comment by borosteve** [comment]
Hi, @Dawe-7 
Would you update me on the progress of this PR please,
Thanks you.

**Comment by Dawe-7** [comment]
> Hi, @Dawe-7
> Would you update me on the progress of this PR please,
> Thanks you.

@borosteve, am reviewing one more article then i'll work on yours.

**Comment by borosteve** [comment]
@Dawe-7 
Aright, thank you for the update.

File: content/articles/html-client-for-web-services-using-dhtml-behavior/index.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+### Distinction between HTML and DHTML
+HTML and DHTML have some of their differences which are as follows:
+1. DHTML language is a collection of different technologies, unlike HTML which is a markup language.
+2. DHTML language needs database connectivity while in HTML it does not need connectivity.
+3. We use DHTML to create dynamic web pages but HTML (a markup language) is employed to construct web pages that are not changing or static.
+4. DHTML does not need processing from the browsers but HTML requires.
+5. We use DHTML to create animation there are also dynamic menus, but no HTML is used.
+6. While using client-side technology, HTML is slower, DHTML sites, on the other hand, are substantially quicker.
+7. Because HTML employs just one language, web pages built with it are simplistic and lack styling, whereas DHTML uses HTML, CSS, and Javascript to build a far better and more attractive webpage.
+8. Only HTML can be used for user-side code, but DHTML is utilized for server-side code.
```
Comment (Dawe-7) [inline content/articles/html-client-for-web-services-using-dhtml-behavior/index.md:54]:
```suggestion
8. HTML can be used in user-side coding whereas DHTML is used for server-side coding.
```

**Review by Dawe-7 — CHANGES_REQUESTED** [review]
@borosteve, welcome to Section's Eng Ed and congratulations on submitting your first article. Your article is an interesting read. To improve the quality of it, I've left you a few suggestions. Consider making those changes. Also, do on the following:
1. Add a prerequisites section - This would be very helpful to readers as they would know what is required before reading.
2. Change the `avator.jpeg` to `avatar.jpg` , `exploler.jpeg` to `explorer.jpg`, `postmethode.jpg` to `postmethod.jpg`. 
3. We do not accept the `.jpeg` image format so use the `.jpg` format, make changes where necessary.

**Comment by Dawe-7** [comment]
@borosteve, any updates on this PR?

**Comment by borosteve** [comment]
Hi @Dawe-7 
Sorry for the delay, I had problems with my laptop.
I have done all the changes you requested.
Thank you.

**Review by Dawe-7 — APPROVED** [review]
@borosteve, kindly go through the changes that I've made. I noticed you didn't implement some of the suggestions i gave. In your future articles, ensure all the suggestions given are implemented to avoid several rounds of revision which leads to article closure. Also, make use of the [hemingway app](https://hemingwayapp.com/) and [grammarly](http://www.grammarly.com/) to check on your article readability. Otherwise, great article. 

@hectorkambow, this article is ready for final review.

cc: @Kayere

**Comment by borosteve** [comment]
Hello, @hectorkambow 
I have made the requested changes.
Thank you.
cc @Dawe-7

**Comment by Dawe-7** [comment]
> Hello, @hectorkambow I have made the requested changes. Thank you. cc @Dawe-7

@hectorkambow, I think the changes seem okay. What do you think?

**Comment by hectorkambow** [comment]
@borosteve 

Please be sure to go through the edits made by your peer reviewer and content moderator (and myself) to improve future articles.
As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published.
Our team of peer reviewers do not provide revisions services at this time in order to review them all in a timely manner. 

All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 
Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

Thank you 🚀 

cc @Dawe-7 please do the same to improve future reviews 

cc @WanjaMIKE please do the same.

**Pull request merged**